ksql-parser/src/main/java/io/confluent/ksql/util/WithClauseUtil.java

```diff
@@ -15,22 +15,15 @@
 
 package io.confluent.ksql.util;
 
-import io.confluent.ksql.parser.tree.Expression;
-import javax.annotation.Nullable;
 import org.apache.commons.lang3.StringUtils;
 
 public final class WithClauseUtil {
 
   private WithClauseUtil() { }
 
-  public static Integer parsePartitions(@Nullable final Expression expression) {
-    if (expression == null) {
```

Comment:
Why is it preferable to move these null checks out of these methods and force each caller to perform the check instead? Seems counterintuitive...

Reply:
I decided it'd be nice to never return null in these methods (more similar to the underlying `parseInt`/`parseShort` methods. I will update the method signature to reflect this, but if you feel strongly I can revert this change